<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form2</class>
 <widget class="QWidget" name="Form2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>407</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>407</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QDateEdit" name="dateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>12</month>
        <day>27</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>12</month>
        <day>27</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>12</month>
        <day>28</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Firm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Route</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QComboBox" name="comboBox_5">
      <item>
       <property name="text">
        <string>Plane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Train</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Car</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ship</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transport</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="3">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Form2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Form2</receiver>
   <slot>addhiketodb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addhiketodb()</slot>
 </slots>
</ui>
